package-overrides/github/BeSite/jQuery.dotdotdot@1.7.4.json

```diff
@@ -0,0 +1,16 @@
+{
+    "main": "jquery.dotdotdot",
+    "dependencies": {
+        "jquery": "*"
+    },
+    "shim": {
+        "jquery.dotdotdot": {
+            "deps": [
+                "jquery"
+            ]
+        }
+    },
+    "directories": {
```

Comment:
We don't encourage `directories.lib` as there is a chance a user may have already installed this and be relying on the existing paths. Best to include `src/js` in all paths for compatibility. The only time we can safely set `directories.lib` is if this is checked into the original package.json file as a PR and bumped into a major release.